slices/libsystemd-shared.yaml

```diff
@@ -7,8 +7,6 @@ slices:
   libs:
     essential:
       - libacl1_libs
-      - libapparmor1_libs
-      - libaudit1_libs
```

Comment:
libsystemd-shared still requires libaudit1 and both shared objects still include references to it. I don't see an upcoming package update in `proposed`, so it's probably worth a comment here.